.github/workflows/locked.yml

```diff
@@ -43,6 +43,10 @@ jobs:
       - name: Test apron regression # skipped by default but CI has apron, so explicitly test group (which ignores skipping -- it's now a feature!)
         run: ruby scripts/update_suite.rb group apron -s
 
+      - name: Test apron regression (Mukherjee et. al  SAS '17 paper') # skipped by default but CI has apron, so explicitly test group (which ignores skipping -- it's now a feature!)
+        if: ${{ matrix.apron }}
```

Comment:
This conditional is unnecessary in the locked workflow, where apron is always enabled via the lockfile. Given that this variable doesn't exist, does this even run?